package.json

```diff
@@ -127,7 +137,8 @@
   },
   "dependencies": {
     "antlr4-c3": "^3.4.1",
-    "antlr4ng": "^3.0.4"
+    "antlr4ng": "^3.0.4",
+    "json5": "^2.2.3"
```

Comment:
I prefer to parse json5 strings with this library to make extraction logic simpler